api/src/Entity/Day.php

```diff
@@ -38,6 +38,7 @@
     ],
     denormalizationContext: ['groups' => ['write']],
     normalizationContext: ['groups' => ['read']],
+    order: ['period.start', 'dayOffset']
```

Comment:
@pmattmann hat in einem anderen PR die Sortierung im Repository hinzugefügt. Ist die eine oder andere Variante besser?

Reply:
#2466